source/includes/fact-call-limitations.rst

```diff
@@ -119,3 +120,24 @@ Use ``.map()`` instead.
 This approach to array sort is often more performant than the
 equivalent unsupported code.
 
+JavaScript Setters 
+~~~~~~~~~~~~~~~~~~
+
+The following JavaScript setter doesn't work: 
```

Comment:
```suggestion
The following JavaScript setter does not work: 
```

a very minor thing, feel free to ignore, but the docs style guide recommends being consistent about contractions within the same page so I thought I'd mention it :)